src/Uno.UI.RuntimeTests.Engine.Library/UnitTestsControl.cs

```diff
@@ -1056,4 +1092,15 @@ private void CopyTestResults(object sender, RoutedEventArgs e)
 
         Clipboard.SetContent(data);
     }
+
+    private void RunOnDispatcher(_DispatcherQueueHandler callback)
```

Comment:
also quick glance at the [uno.ui winui-autoconvert source](https://github.com/unoplatform/uno/blob/generated/master/winui-autoconvert/src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTest/UnitTestsControl.cs), we are still calling the Dispatcher
this could be problematic, unless we are not running that on winui-desktop

Reply:
Makes sense, I'll take a look as part of https://github.com/unoplatform/uno/pull/10008